### PR TITLE
Added more section in details components

### PR DESCRIPTION
**Pull request by animeshk874**

### CODE REVIEW

File: src/components/details/details.component.js [hunk 1]
```diff
@@ -20,22 +20,29 @@ export default function Details() {
   const [details, setDetails] = useState(null);
 
   useEffect(() => {
-    if(!dataStructureKey){
+    if (!dataStructureKey) {
       return;
     }
     fetch(`/data/data-structure-information/${dataStructureKey}.json`)
```
Comment (surya76657) [inline src/components/details/details.component.js:27]: Can you clear the data before fetching the query...

Comment (surya76657) [reply src/components/details/details.component.js:27]: Anyway the query params is changed... And data is showing wrong

File: src/components/details/details.component.js [hunk 2]
```diff
@@ -91,6 +107,63 @@ export default function Details() {
                                 </div>
                               </>
                             }
+                            {
+                              operation.parameters && operation.parameters.length ?
+                                <>
```
Comment (surya76657) [inline src/components/details/details.component.js]: Use "< Fragment >" instead of <>

**Review by surya76657 — APPROVED** [review]

File: src/components/details/details.component.js (hunk 1 above)
Comment (animeshk874) [reply src/components/details/details.component.js:27]: Done

File: src/components/details/details.component.js (hunk 2 above)
Comment (animeshk874) [reply src/components/details/details.component.js]: Done

**Pull request merged**